terraspace/README.md

```diff
@@ -0,0 +1,19 @@
+# Terraspace Project
+
+This is a Terraspace project. It contains code to provision Cloud infrastructure built with [Terraform](https://www.terraform.io/) and the [Terraspace Framework](https://terraspace.cloud/).
+
```

Comment:
Worth updating this readme in our subsequent PRs once we have our specific stacks and modules to talk about

Reply:
Agree!